Mathlib/Init/Data/Bool/Lemmas.lean

```diff
@@ -47,8 +47,7 @@ namespace Bool
 theorem cond_self.{u} {α : Type u} (b : Bool) (a : α) : cond b a a = a := by cases b <;> rfl
 #align cond_a_a Bool.cond_self
 
-@[simp]
-theorem xor_self (b : Bool) : xor b b = false := by cases b <;> simp
+attribute [simp] xor_self
 #align bxor_self Bool.xor_self
 
 @[simp]
```

Comment:
Std called this `Bool.xor_true` lemma `Bool.xor_true_right`; @fgdorais, is there a reason you choose to rename the mathlib version? Or can we change the name in Std to match?

Reply:
This is not a rename, Std should have both as aliases. It's not there because there was a debate at some point whether `xor` should have an infix notation and I wasn't sure what the implications of that would be. (Mario clarified elsewhere but I didn't remember to fix it here.) I will add that to the follow-up PR.

Reply:
Fixed in std4#329 and #8005